pkgs/shells/bash/ble.sh/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, git, fetchgit }:
+
+stdenv.mkDerivation {
+  name = "ble.sh";
+  version = "0.3.3";
+
+  nativeBuildInputs = [ git ];
+
+  src = fetchgit {
+    url = "https://github.com/akinomyoga/ble.sh.git";
+    rev = "refs/tags/v0.3.3";
```

Comment:
Use `fetchFromGitHub`.

Reply:
v0.3.3 does not need it, but on latest unstable ble.sh needs .git directory. For this reason, on the upcoming new version `fetchgit`'s `leaveDotGit` will be needed.
I'll add `leaveDotGit` on this version of derivation too.

Reply:
Then fetchFromGitHub must be used with leaveDotGit. Also substitute the version number

Reply:
I didn't notice fetchFromGitHub has leaveDotGit! My bad, I should have read the docs more carefully. Thanks, I'll update this